### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by humancalico**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-# sophosd
-Daemon for connecting to Sophos written in Rust
+# Sophosd
+
+Daemon for connecting to Sophos written in Rust.
+
+# Installation and Usage
+
+The [Rust compiler and Cargo package manager](https://www.rust-lang.org/learn/get-started) are needed. You may also need to install an OpenSSL development package - this is `libssl-dev` on Ubuntu and `openssl-devel` on Fedora.
```
Comment (ryzokuken) [inline README.md]: Why is it required? Is it a Rust dependency?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+Clone this repository and `cd` to base directory where `Cargo.toml` is located and use the `cargo` commands from here. For example to use use sophos  by using `cargo run` just type:
+
+```$ cargo run <username> <password> ```
+
+Alternatively you can also create a binary Debian package `.deb` file using the cargo-deb command. The resulting package would be placed in `/target/debian/`.
+
+If you have installed sophosd using the `.deb` file you can use it in the command line by typing in:
+
+```$ sophsosd <username> <password>```
```
Comment (ryzokuken) [inline README.md]: Could you please break this section up maybe?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
-# sophosd
-Daemon for connecting to Sophos written in Rust
+# Sophosd
```
Comment (ryzokuken) [inline README.md]: could you maybe keep it lowercase and make it monospace? Like `sophosd`.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
+
+```cargo install cargo-deb```
+
+Now, run this in your Cargo project directory
+
+```cargo deb```
+
+The resulting package would be placed in `/target/debian/`. Install this and now you can use `sophosd` by typing
+
+```sophsosd <username> <password>```
```
Comment (ryzokuken) [inline README.md]: This isn't required. Normal users almost never really need to build a package, it's fine to build a binary using `cargo build` and just run the binary.

**Comment by akshat2654** [comment]
In order to build software which requires the _OpenSSL general-purpose library_, you must first install the _development instance of the OpenSSL library_. 
[source](https://help.ubuntu.com/community/OpenSSL)

**Comment by akshat2654** [comment]
> P.S. you never mentioned why you needed OpenSSL here. I never realized since I had OpenSSL installed anyway.

In order to build software which requires the _OpenSSL general-purpose library_, you must first install the _development instance of the OpenSSL library_
[source](https://help.ubuntu.com/community/OpenSSL#Practical_OpenSSL_Usage)

**Comment by ryzokuken** [comment]
@akshat2654 no, that's not what I meant, I don't "recall" using OpenSSL anywhere, that's why I asked.

**Comment by akshat2654** [comment]
It's in the `Cargo.lock` file

**Comment by ryzokuken** [comment]
@akshat2654 This is only an issue for linux users, I see: https://github.com/seanmonstar/reqwest/blob/87a09322d6fef48e7a7c79a854acaf08bb16f606/README.md#requirements

**Comment by ryzokuken** [comment]
Closing since #5 is now merged.

**Pull request closed without merging**